src/main/java/art/heredium/domain/company/model/dto/request/CompanyMembershipRegistrationRequest.java

```diff
@@ -0,0 +1,32 @@
+package art.heredium.domain.company.model.dto.request;
+
+import java.time.LocalDate;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@Getter
+@Setter
+public class CompanyMembershipRegistrationRequest {
+  @NotBlank private String title;
+
+  @NotBlank
+  @JsonProperty("email_or_phone")
+  private String emailOrPhone;
+
+  @NotNull
+  @JsonProperty("start_date")
+  private LocalDate startDate;
+
+  @NotNull @Positive private Integer price;
```

Comment:
```suggestion
  @NotNull @Positive private Long price;
```
Bỏ hết @JsonProperty

Reply:
I updated in https://github.com/Incuvers-IT/heredium-BE/pull/93/commits/c482a5a6f5e68a82880a2b490c7f92cdd484bbfe